src/bun.js/bindings/JSSinkLookupTable.h

```diff
@@ -119,11 +119,11 @@ static const struct HashTableValue JSFileSinkPrototypeTableValues[7] = {
 static const struct HashTable JSFileSinkPrototypeTable =
     { 7, 15, false, nullptr, JSFileSinkPrototypeTableValues, JSFileSinkPrototypeTableIndex };
 
+} // namespace JSC
 
+#include "Lookup.h"
```

Comment:
something went wrong here, the imports should've been stripped

Reply:
weird, looks like `make codegen` did not complete (probably because I hit some key)